### PR TITLE
Cleanup/images

### PR DESCRIPTION
**Pull request by deepakprabhakara**

### CODE REVIEW

File: guides/jackson/integrations/bubble.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Next step is to configure the plugin. Make sure you add the following values in
 
 - `Client Secret Verifier`: This is optional. If you have configured a client secret verifier in SAML Jackson, you can add it here. If you have not configured a client secret verifier, you can leave this field blank.
 
-![Configure the plugin](/img/bubble-io/step-1.png)
+![Configure the plugin](./bubble/step-1.png)
```
Comment (schalkneethling) [inline guides/jackson/integrations/bubble.md:38]: Should we copy these to `images`?

File: src/pages/saas-producthunt.js [hunk 2]
```diff
@@ -57,13 +57,13 @@ const ProductHuntSaas = () => {
...
-        title="Founder Dub"
-        avatar="/img/product-hunt/steven-tey.jpeg"
+      <FeaturedQuote
+        personName="Steven Tey"
+        personRole="Founder Dub"
+        pictureSrc="/images/success-stories/steven-tey-dub"
       >
-        Boxy is an absolute godsend 🔥
-      </Quote>
+        🔥 Boxy is an absolute godsend!
```
Comment (schalkneethling) [inline src/pages/saas-producthunt.js:65]:
```suggestion
        🔥 BoxyHQ is an absolute godsend!
```

File: success-stories/2023-11-27-safeguarding-trust-boxyhq-sso-enhances-dubs-security.md [hunk 3]
```diff
@@ -2,25 +2,25 @@
...
 
 In the fast-paced digital landscape of today, security and efficiency are paramount. Dub, a forward-thinking organization, recognized the need to streamline and strengthen their user authentication process. The motivation behind implementing [Enterprise Single Sign-On (SSO)](https://github.com/boxyhq/jackson) was to simplify user access while bolstering security measures. Dub aimed to provide a seamless experience for their clients and employees, eliminating the hassle of multiple logins and passwords.
 
-<Quote author="Steven Tey" title="Founder Dub" avatar="/img/success-stories/steven-tey-dub.jpeg">
-  🔥 Boxy is an absolute godsend!
-</Quote>
+<SignupCTA campaign="success-story-dub" />
+
+<FeaturedQuote personName="Steven Tey" personRole="Founder Dub" pictureSrc="/images/success-stories/steven-tey-dub">
+ 🔥 Boxy is an absolute godsend!
```
Comment (schalkneethling) [inline success-stories/2023-11-27-safeguarding-trust-boxyhq-sso-enhances-dubs-security.md:22]:
```suggestion
 🔥 BoxyHQ is an absolute godsend!
```

**Review by schalkneethling — COMMENTED** [review]
Thanks, @deepakprabhakara - I did a first pass over all the changes and made two or three suggestions. I will pull this down a do some manual testing as well.

File: guides/jackson/integrations/bubble.md (hunk 1 above)
Comment (deepakprabhakara) [reply guides/jackson/integrations/bubble.md:38]: Moving ahead we should keep assets with markdown, docusaurus will handle it automatically.

**Review by schalkneethling — APPROVED** [review]

**Pull request merged**